### DISCUSSION TITLE
Magma AGW 1.7 helmcharts

### DISCUSSION THREAD

**Discussion by palyogendra** [magma/magma#12487] [General]
Hi,

Now that Magma 1.7 release is pretty much done, where we can find the helmcharts for AGW w.r.t Magma release 1.7

Appreciate your support and time.

Best Regards, Yogen

**Comment by maaathieu-x** [comment]
+1
lte-orc8r-0.2.6.tgz has been publisehd on 04/02/2022, same date as 1.7 release, so maybe it's the same. 

Mat

**Reply by palyogendra** [reply]
I don't think they're the same, lte-orc8r-0.2.6.tgz charts and orc8r-1.5.27.tgz both are part of Magma controller and the AGW is data plane.

-Yogen
